src/components/ReleaseAnnouncement/ReleaseAnnouncement.module.css

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2024 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.content {
+  /* 320px + 20px padding left + 20px padding right */
+  max-inline-size: 320px;
+  border-radius: var(--cpd-space-3x);
+  background: var(--cpd-color-bg-action-primary-rest);
+  padding: var(--cpd-space-4x) var(--cpd-space-5x);
```

Comment:
The designs say this is supposed to be be `var(--cpd-space-5x) var(--cpd-space-6x)`